torch/lib/TH/THGeneral.h.in

```diff
@@ -42,8 +42,14 @@
 typedef void (*THErrorHandlerFunction)(const char *msg, void *data);
 typedef void (*THArgErrorHandlerFunction)(int argNumber, const char *msg, void *data);
 
+#define TH_DESC_BUFF_LEN 64
```

Comment:
I don't think it's necessary - it's just 64B of stack space. Better to have a margin. It doesn't cost us anything